setup.py

```diff
@@ -7,7 +7,11 @@
     "pyyaml >= 5.1.1",
     "requests >= 2.22.0",
 ]
-dev_requirements = ["black >= 19.10b0", "ipdb >= 0.13.2", "pre-commit >= 1.20.0"]
+dev_requirements = [
```

Comment:
Do we still need this file `setup.py`? Or can we delete, since Poetry is taking care of it?

Reply:
Yes. We'll removed.